src/ui/marker.js

```diff
@@ -446,41 +452,59 @@ export default class Marker extends Evented {
             position.y >= 0 && position.y < tr.height;
     }
 
-    _update(e?: {type: 'move' | 'moveend'}) {
-        if (!this._map) return;
+    _updateDOM(pos: Point) {
```

Comment:
Great work @malekeym!
One more tiny nit: Why does this function take in the position as an argument? Given it's directly accessing class variables like `this._anchor`, I think that it would be more consistent and prevent potential confusion to either remove this argument, or else pass in the other variables as arguments.

Reply:
@SnailBones Done!

Reply:
Looks great!